feature-libs/my-account/organization/src/components/cost-center/edit/cost-center-edit.component.ts

```diff
@@ -18,13 +19,21 @@ import { CostCenterFormService } from '../form/cost-center-form.service';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class CostCenterEditComponent {
-  protected code$: Observable<string> = this.activatedRoute.parent.params.pipe(
-    map((routingData) => routingData['code'])
-  );
+  /**
+   * The code of the current cost center
+   */
+  protected code$ = this.currentCostCenterService.code$;
 
-  protected costCenter$: Observable<CostCenter> = this.code$.pipe(
+  /**
+   * The model of the current cost center.
+   *
+   * It reloads the model when the code of the current cost center changes.
+   */
+  protected costCenter$: Observable<
+    CostCenter
+  > = this.currentCostCenterService.code$.pipe(
     tap((code) => this.costCenterService.load(code)),
-    switchMap((code) => this.costCenterService.get(code)),
+    switchMapTo(this.currentCostCenterService.model$),
```

Comment:
As discussed, we can use facade here, as it looks more natural in case we already have it.